packages-top/xsschema/src/to-json-schema/vendors/sury.ts

```diff
@@ -0,0 +1,8 @@
+import type { Schema } from 'sury'
+
+import { toJSONSchema } from 'sury'
```

Comment:
We should use tryImport(), otherwise it will throw an error.

Reply:
True, missed this part

Reply:
Fixed